Source/WebCore/editing/cocoa/DictionaryLookup.mm

```diff
@@ -313,7 +313,7 @@ static bool canCreateRevealItems()
     if (position.isNull())
         position = firstPositionInOrBeforeNode(node);
 
-    RefPtr focusedOrMainFrame = frame->page()->checkedFocusController()->focusedOrMainFrame();
+    RefPtr focusedOrMainFrame = frame->page()->focusController().focusedOrMainFrame();
```

Comment:
protectedPage?

Reply:
The four instances that you flagged are all okay I think as focusedOrMainFrame is trivial. This is also demonstrated by `DictionaryLookup.mm` now being safe as far as UncountedCallArgsChecker goes.

But per our offline discussion you did make me realize that there was a small safety regression for a couple of cases where a non-trivial method followed focusController(). (Though these were already unsafe.)

And additionally we had too much protection in a large number of cases as the majority of callers is interested in focusedOrMainFrame() and thus no longer needed a protected page.